src/viur_cli/conf.py

```diff
@@ -82,7 +82,21 @@ def delete(self):
         except:
             raise click.ClickException(click.style(f"{configname} not found", fg="red"))
 
+    def find_key(self, dictionary, target_key, target, level=1):
+        for key, value in dictionary.items():
+            if key == target_key:
+                self[target][target_key] = value
+                return
+            if isinstance(value, dict):
+                self.find_key(value, target_key, target, level + 1)
```

Comment:
I think this might be faster!
```suggestion
        if value := dictionary.get(target_key):
           self[target][target_key] = value
           return
           
        for value in dictionary.values():
            if isinstance(value, dict):
                self.find_key(value, target_key, target, level + 1)
```